### PR TITLE
Fix missing parameters of execution function of OvnCommand

### PR DESCRIPTION
**Pull request by fyuan1316**
# add the missing parameters, so we can execute ovnCommand directly.

Before:
---
ocmd, _ := ovndbapi.LSAdd("ls1")
ovndbapi.Execute(ocmd)

After:
---
ocmd, _ := ovndbapi.LSAdd("ls1")
ocmd.Execute()

### CODE REVIEW

**Comment by fyuan1316** [comment]
@hzhou8  @noah8713 
It looks like the parameters are forgotten, if this is the original design, please correct me  ：）

**Comment by fyuan1316** [comment]
add example and some description ：）

**Comment by noah8713** [comment]
@fyuan1316 : Did CI pass for this change?

File: client.go [hunk 1]
```diff
@@ -176,6 +176,8 @@ type Client interface {
 	Close() error
 }
 
+var _ Client = &ovndb{}
```
Comment (hzhou8) [inline client.go]: Why this change?

Comment (fyuan1316) [reply client.go]: Just to ensure that the `ovndb` struct  implements the methods defined by the `Client` interface

**Comment by noah8713** [comment]
Thanks for the explaination!

**Comment by hzhou8** [comment]
This PR should be squashed before merge.

**Comment by noah8713** [comment]
This PR had two commits in same PR. Merge didn't complain for rebase so I merged it instead of squash.

**Comment by hzhou8** [comment]
@noah8713 The second commit is a fix of the previous commit (a revision based on review comment), which is why they should be squashed. In another words, they belong to same change, and the principles is one commit for one change.

**Comment by noah8713** [comment]
@hzhou8 : I just checked the commits. Since commit https://github.com/eBay/go-ovn/pull/79/commits/40b307c93fbdf702fba1698989bf46b3d814cb0d is the one on top of https://github.com/eBay/go-ovn/pull/79/commits/0f9821be95ba27b9273b028633a249d23cbe6b25 , review comments code got addressed fine. So ideally what you are proposing is user should be sending one PR for each commit instead of two commits in same PR. Do you want me to revert the merge? We can ask user to send one PR with both two commits squashed into one.
